instrumentation-core/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/exporters/ExporterAutoConfiguration.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.instrumentation.spring.autoconfigure.exporters;
+
+import io.opentelemetry.exporters.logging.LoggingSpanExporter;
+import io.opentelemetry.instrumentation.spring.autoconfigure.exporters.jaeger.JaegerExporterFactory;
+import io.opentelemetry.instrumentation.spring.autoconfigure.exporters.jaeger.JaegerExporterProperties;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableConfigurationProperties({ExporterProperties.class, JaegerExporterProperties.class})
+@ConditionalOnProperty(prefix = "opentelemetry.exporter", name = "enabled", matchIfMissing = true)
+public class ExporterAutoConfiguration {
+
+  @Bean
+  @Autowired
+  @ConditionalOnMissingBean
+  public SpanExporter otelSpanExporter(
+      ExporterProperties exporterProperties, JaegerExporterProperties jaegerExporterProperties) {
+
+    switch (exporterProperties.getExporterType()) {
+      case JAEGER:
+        return JaegerExporterFactory.createJaegerExporter(jaegerExporterProperties);
+      case ZIPKIN:
```

Comment:
[Question] Is it worth debug logging some kind of "using standard exporter" message? 
Alternatively, it may be worth explicitly indicating which exporters are not ready yet (and maybe adding a link to the FR), vs which ones are intended forever to go to the standard exporter.

Reply:
I removed this enum